src/generate/apiModel.test.ts

```diff
@@ -102,6 +112,46 @@ describe("ApiModel tests", () => {
       "Cannot update the name before the model is loaded"
     );
   });
+
+  it("uses title for class name if custom field is not set", async () => {
+    const api = new ApiModel("VALID", path.dirname(validRamlFile));
+    await api.init();
+    expect(api.name).to.deep.equal(new Name("Shop API"));
```

Comment:
Why did you add 'deep' here, is it necessary?

Reply:
I believe that `deep` recursively goes through the object to ensure that all properties are equal and structure of the object is the same. I just tested `expect(api.name).to.equal(new Name("Shop API"));` without deep and the unit test fails